testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -3747,19 +3747,19 @@ QUnit.module('searchEnabled', moduleSetup, () => {
 
     QUnit.test('search was work if acceptCustomValue is set to true', function(assert) {
         const $element = $('#tagBox').dxTagBox({
-            dataSource: ['item 1', 'element 1', 'item 2'],
+            dataSource: ['item 1', 'element 1', 'item 2', 'item 3'],
```

Comment:
Why should we change this test?

Reply:
https://github.com/DevExpress/DevExtreme/pull/22772/commits/87a7c60cf7177115eeeabf74aeb88d3ca9866bd2